### DISCUSSION TITLE
need help about action !! thanks

### DISCUSSION THREAD

**Discussion by skydig** [leptos-rs/leptos#2686] [General]
I have created one action and dispatch when button is clicked, how can I know the action is finished or not?
because I want to reload something, when the action finished.
Thanks.

**Comment by gbj** [comment]
Use the action's [`value`](https://docs.rs/leptos/latest/leptos/struct.Action.html#method.value) or [`version`](https://docs.rs/leptos/latest/leptos/struct.Action.html#method.version) signals.

**Reply by skydig** [reply]
thanks, which example I can reference?
